### PR TITLE
Remove package-lock.json and require node 14

### PR DESCRIPTION
**Pull request by goplayoutside3**
Resolves Issue #11

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:10
+FROM node:14
 WORKDIR /src
 
 # PostgreSQL 9.5 apt package setup
```
Comment (srallen) [inline Dockerfile]: This comment is outdated now.
```suggestion
```

**Review by srallen — DISMISSED** [review]
LGTM and docker is working locally now.

**Comment by srallen** [comment]
The DB itself I think still needs some upgrade related maintenance, I do see in the logs, but perhaps for a separate PR?:

```
postgres_1  | 2021-04-08 22:02:27.036 UTC [1] FATAL:  database files are incompatible with server
postgres_1  | 2021-04-08 22:02:27.036 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 11.11 (Debian 11.11-1.pgdg90+1).
```

**Comment by zwolf** [comment]
^^ That'd be me. I updated the required version of Postgres without updating the schema file itself. I'll fix that and this can be rebased.

my b zach--

**Review by srallen — APPROVED** [review]
As we discussed on the call, some readme updates would be helpful to include running `yarn install` first before running your container, and `-p` to specify the port on the yarn start command. Some future updates to the dockerfile could be considered as well as ignoring the node_modules folder from your local system.

**Pull request merged**